cfgov/v1/static/js/admin/ask_cfpb_tips.js

```diff
@@ -11,6 +11,8 @@
       populateToolbar: function (toolbar) {
         if (window.location.href.indexOf('ask_cfpb') > -1) {
           var buttonset,
+          baseClass = "answer-module",
+          textInputClass = 'richtext',
           _this = this;
           buttonset = jQuery("<span class=\"" + this.widgetName + "\"></span>");
```

Comment:
ok, make sense. I didn't realize it was for the admin. We place ours in https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/templates/wagtailadmin/js/table-block.js